pandas/tests/frame/methods/test_explode.py

```diff
@@ -9,13 +9,34 @@ def test_error():
     df = pd.DataFrame(
         {"A": pd.Series([[0, 1, 2], np.nan, [], (3, 4)], index=list("abcd")), "B": 1}
     )
-    with pytest.raises(ValueError, match="column must be a scalar"):
+    with pytest.raises(
```

Comment:
this test is getting big, can you split into multiple ones (ok to rename the original), parameterize is good as well if posilbe